R/print.data.table.R

```diff
@@ -145,3 +145,6 @@ shouldPrint = function(x) {
 # for removing the head (column names) of matrix output entirely,
 #   as opposed to printing a blank line, for excluding col.names per PR #1483
 cut_top = function(x) cat(capture.output(x)[-1L], sep = '\n')
+
+# FR #2591 - format.data.table issue with columns of class "formula"
+is.formula <- function(x) class(x) == "formula"
```

Comment:
^shouldn't this be `inherits(x, 'formula')`?

Reply:
Yes. Good spot. I'm about to merge so I'll do it quickly.